bundles/org.openhab.binding.somfytahoma/src/main/java/org/openhab/binding/somfytahoma/internal/handler/SomfyTahomaHitachiATWHZHandler.java

```diff
@@ -0,0 +1,67 @@
+/**
+ * Copyright (c) 2010-2022 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.somfytahoma.internal.handler;
+
+import static org.openhab.binding.somfytahoma.internal.SomfyTahomaBindingConstants.*;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.openhab.core.library.types.StringType;
+import org.openhab.core.thing.ChannelUID;
+import org.openhab.core.thing.Thing;
+import org.openhab.core.types.Command;
+
+/**
+ * The {@link SomfyTahomaHitachiATWHZHandler} is responsible for handling commands,
+ * which are sent to one of the channels of the Hitachi Air To Water Heating Zone thing.
+ *
+ * @author Ondrej Pecta - Initial contribution
+ */
+@NonNullByDefault
+public class SomfyTahomaHitachiATWHZHandler extends SomfyTahomaBaseThingHandler {
+
+    public SomfyTahomaHitachiATWHZHandler(Thing thing) {
+        super(thing);
+        stateNames.put(ZONE_MODE, "modbus:AutoManuModeZone1State");
+        stateNames.put(CIRCUIT_CONTROL, "modbus:ControlCircuit1State");
+        stateNames.put(CIRCUIT_STATUS, "modbus:StatusCircuit1State");
+        stateNames.put(YUTAKI_TARGET_MODE, "modbus:YutakiTargetModeState");
+        stateNames.put(YUTAKI_MODE, "modbus:YutakiModeState");
+        stateNames.put(HOLIDAY_MODE, "modbus:HolidayModeZone1State");
+        stateNames.put(ALARM_NUMBER, "modbus:AlarmNumberState");
+    }
+
+    @Override
+    public void handleCommand(ChannelUID channelUID, Command command) {
+        super.handleCommand(channelUID, command);
+        if (command instanceof StringType) {
```

Comment:
Please catch the REFRESH command in your 3 new handlers like you did in other handlers.

Reply:
OK